dkms.in

```diff
@@ -2499,6 +2499,11 @@ done
 
 # Sanity checking
 
+# The <(cmd) idiom does not work if /proc is not mounted
+if [ ! -e <(echo) ]; then
+    warn $"dkms will not function properly if /proc is not mounted."
+fi
+
```

Comment:
How did you hit this? Can you add some context in the commit message of such setups and the (some) of the issues you've seen.

Reply:
This seems to be tripping up the Ubuntu VM CI. It's possible that permutation is broken and it has been green by a fluke.

But for the time being, let's pull out this patch and land the PR. It's looking good to land.

Reply:
OK, I'll move that to some 'postponed' branch.